app/views/subjects/index.html.haml

```diff
@@ -1,4 +1,8 @@
 %h1 Subjects
 %ul
+  - if Subject.count == 0
+    %li= 'No Subjects to Display.'
```

Comment:
This is true :P. But once we create our first subject this is deal code forever.